docs/platform/visual/index.md

```diff
@@ -34,21 +33,17 @@ Visualizations are implemented on top of the Pentaho Core, Type and Data JavaScr
   as well as with other client-side component frameworks.
 - Using [Core APIs]()
   provides visualizations with features such as localization, theming and registration.
-  
-The use of [Pentaho Platform JavaScript APIs]({{site.refDocsUrlPattern | replace: '$', 'pentaho'}}) for anything not 
-directly related with the visualization concept, 
-ensures developers can reuse knowledge that they gain in other components of the Pentaho suite 
-to build visualizations and vice-versa.
```

Comment:
This recalls the central position of the Platform JavaScript APIs. It says to developers that the knowledge they gain in these can be capitalized in multiple components of the Pentaho suite. When you learn these APIs while developing vizs, you'll gain knowledge that you'll be able to use when developing CDF dashboards, and vice-versa. When you learn these APIs while developing vizs, you'll gain knowledge that you'll be able to use when developing vizs.

It also frames the strategical positioning of the Visualization API, of leaving out any abstractions not strictly related with visualization.

I think this is relevant. Maybe someone can provide a clearer formulation.